boards/arm/bl5340_dvk/Kconfig.defconfig

```diff
@@ -83,3 +80,87 @@ endif # BOARD_BL5340_DVK_CPUAPP || BOARD_BL5340_DVK_CPUAPP_NS
 
 config BOARD
 	default "bl5340_dvk_cpunet" if BOARD_BL5340_DVK_CPUNET
+
+config IPM_NRFX
+	default IPM
+
+config MBOX_NRFX_IPC
+	default MBOX
+
+if BOARD_BL5340_DVK_CPUAPP || BOARD_BL5340_DVK_CPUAPP_NS
+
+choice BT_HCI_BUS_TYPE
+	default BT_RPMSG if BT
+endchoice
+
+config HEAP_MEM_POOL_SIZE
+	default 4096 if BT_RPMSG
+
+config BT_HCI_VS
+	default y if BT
+
+# 2.8" TFT Touch Shield for Arduino w/Capacitive Touch Screen Module
+# is mounted to the DVK using standoffs.
+# https://www.buydisplay.com/2-8-inch-tft-touch-shield-for-arduino-w-capacitive-touch-screen-module
+if DISPLAY
+
+if KSCAN
+
+config INPUT_FT5336
+	default y
+
+config INPUT_FT5336_INTERRUPT
+	default y
+
+endif # KSCAN
+
+if LVGL
+
+config LV_Z_VDB_SIZE
+	default 64
+
+config LV_Z_BITS_PER_PIXEL
+	default 16
+
+choice LV_COLOR_DEPTH
+	default LV_COLOR_DEPTH_16
+endchoice
+
+config LV_COLOR_16_SWAP
+	default y
+
+config KSCAN
+	default y
+
+config INPUT
+	default y
+
+config LV_Z_POINTER_KSCAN
+	default y
```

Comment:
input has replaced kscan - @fabiobaltieri for comment on this

Reply:
It's in progress, LVGL still takes kscan events. Will write an lvgl pointer driver using the input APIs down the road but for now this is still the way to go.

More importantly:

```
# 2.8" TFT Touch Shield for Arduino w/Capacitive Touch Screen Module
```

If the display is on a shield this should not go in the board dts at all. Instead the board should define an `arduino-header-r3` node and then the display can be enabled with the generic shield code that already exists in `boards/shields/buydisplay_2_8_tft_touch_arduino`.

Reply:
Our board has the display attached already and takes work for a user to remove it. See here:
https://www.lairdconnect.com/wireless-modules/bluetooth-modules/bluetooth-5-modules/bl5340-series-multi-core-bluetooth-52-802154-nfc-modules

Reply:
Ok then this makes sense.